Automattic/wp-calypso
# Update site disconnected filter key

Updating the key for the site disconnected filter to match the filter name and action

#### Proposed Changes

* This PR updates the query param for the site disconnected filter from `site_down` to `site_disconnected` to match the filter action

#### Testing Instructions



* Run `git checkout update/update-jetpack-pro-dashboard-filters`
* Run `yarn-start-jetpack-cloud`
* Visit the dashboard at `http://jetpack.cloud.localhost:3000/dashboard`
* Use the Issue filters to filter by Site disconnected
* Ensure that the filter is applied and the query param used is `site_disconnected` instead of `site_down`
* Ensure that the results are consistent with the filter action

#### Pre-merge Checklist





Related to #
